.github/workflows/test-go.yml

```diff
@@ -225,11 +296,6 @@ jobs:
             kv/data/github/${{ github.repository }}/github-token username-and-token | github-token;
             kv/data/github/${{ github.repository }}/license license_1 | VAULT_LICENSE_CI;
             kv/data/github/${{ github.repository }}/license license_2 | VAULT_LICENSE_2;
-            kv/data/github/${{ github.repository }}/hcp-link HCP_API_ADDRESS;
```

Comment:
We remove getting and setting these env vars. That sems to be okay as later one is commented out to prevent executing these tests.